zerver/lib/markdown/api_arguments_table_generator.py

```diff
@@ -125,6 +125,9 @@ def render_table(self, arguments: Sequence[Mapping[str, Any]]) -> List[str]:
             example = ""
             if "example" in argument:
                 example = argument["example"]
+                # Converting camel-case boolean to lower case.
+                if str(example) == "True" or str(example) == "False":
+                    example = str(example).lower()
```

Comment:
Using `json.dumps` on `example` without `content` and `application/json` in `argument` adds quotation marks to word(string) examples. so this is the simplest solution I found for this as it only addresses boolean values.

Reply:
I hadn't seen your comment yet; but I think those quotes are net helpful for readability; I think users are unlikely to double-quote strings, and having the quotation marks makes them look more like strings.

Reply:
Yes, agreed!.  I wanted to address the boolean values alone as the issue was raised specifically for that and intentionally avoided json dumps as it added quotes. As you said I guess having quotes for strings is helpful and matches with the 
`defaults to` format as it uses Json dumps too  :) !